sdk/nodejs/resource.ts

```diff
@@ -466,6 +482,10 @@ export interface ResourceOptions {
      * An optional customTimeouts configuration block.
      */
     customTimeouts?: CustomTimeouts;
+    /**
+     * Optional list of transformations to apply to this resource during construction.
+     */
+    transformations?: ResourceTransformation[];
```

Comment:
document what thsi means if there are multiple transformations specified.   document hte parent.child relationship.

Reply:
this can, of course, wait until we've hammered out what we actually want.

Reply:
Fixed.